src/derivers/bip32.ts

```diff
@@ -127,6 +114,45 @@ type GenerateKeyArgs = {
   secretExtension: string | Buffer;
 };
 
+/**
+ * Get a BigInt from a byte array.
+ *
+ * @param bytes - The byte array to get the BigInt for.
+ * @returns The byte array as BigInt.
+ */
+function bytesToNumber(bytes: Uint8Array): bigint {
```

Comment:
Move to `utils.ts`?

Reply:
It's not used outside of this file. I can move it if you prefer, though.

Reply:
imo its good practice to place very re-usable code that like in `utils`. But ultimately up to you 😄